Sources/VariadicsGenerator/VariadicsGenerator.swift

```diff
@@ -783,4 +789,38 @@ struct VariadicsGenerator: ParsableCommand {
 
       """)
   }
+  
+  func emitCompactingCapture(arity: Int) {
+    output("""
+      \(defaultAvailableAttr)
```

Comment:
Availability?

Reply:
I thought that's what this was for — does `Variadics.swift` not have the correct availability annotations?

Reply:
I am blind